probe-rs/src/probe/daplink/mod.rs

```diff
@@ -357,6 +358,36 @@ impl DebugProbe for DAPLink {
         Ok(())
     }
 
+    fn target_reset_assert(&mut self) -> Result<(), DebugProbeError> {
+        let mut mask = Pins(0);
+        mask.set_nreset(true);
+        mask.set_ntrst(true);
+        let request = SWJPinsRequest {
+            output: mask,
+            select: Pins(0x00),
+            wait: 0,
+        };
+        commands::send_command(&mut self.device, request).map(|v: SWJPinsResponse| {
+            log::info!("Pin response: {:?}", v);
+        })?;
+        Ok(())
+    }
+
+    fn target_reset_deassert(&mut self) -> Result<(), DebugProbeError> {
+        let mut mask = Pins(0);
+        mask.set_nreset(true);
+        mask.set_ntrst(true);
+        let request = SWJPinsRequest {
+            output: mask,
+            select: Pins(0xFF),
```

Comment:
As above.

```suggestion
            output: Pins(0xFF),
            select: mask,
```

Reply:
Good catch! I've added a helper struct to prevent this kind of error in the future. See `SWJPinsRequestBuilder`.